dubbo-common/src/main/java/org/apache/dubbo/common/utils/NetUtils.java

```diff
@@ -118,6 +118,20 @@ public synchronized static int getAvailablePort(int port) {
         return port;
     }
 
+    /**
+     * Check the port whether is in use in os
+     * @param port

```

Comment:
Please improve the javadoc so `@param port` and `@return` is complete

I suggest to complete it with:
`@param port   the port to check if in use`
`@return is the given port in use or not`

Reply:
done